src/cb_admin/pages/ConfirmPassword.js

```diff
@@ -25,17 +27,24 @@ export default class ConfirmPassword extends React.Component {
 
     this.state = {
       errors: {},
+      email: null,
     };
   }
 
+  componentDidMount() {
+    CbAdmin.get()
+      .then(res => this.setState({ email: pathOr('', ['email'])(ResponseUtils.getResponse(res)) }))
```

Comment:
Wouldn't it be simpler to build this into the definition of `getResponse`, like in my example? It's likely to be a utility function that we use in many other places going forward.